neuralmonkey/encoders/cnn_encoder.py

```diff
@@ -52,31 +52,27 @@ def __init__(self,
         """Initialize a convolutional network for image processing.
 
         Args:
-            convolutions (list): Configuration convolutional layers. It is a
-                list of tripplets of integers where the values are: size of the
+            convolutions: Configuration of convolutional layers. It is a list
+                of triplets of integers where the values are: size of the
                 convolutional window, number of convolutional filters, and size
                 of max-pooling window. If the max-pooling size is set to None,
                 no pooling is performed.
             data_id: Identifier of the data series in the dataset.
             image_height: Height of the input image in pixels.
-            image_width: Width of the images (padded)
+            image_width: Width of the image.
             pixel_dim: Number of color channels in the input images.
             batch_normalization: Flag whether the batch normalization
                 should be used between the convolutional layers.
             local_response_normalization: Flag whether to use local
                 response normalization between the convolutional layers.
-            dropout_placeholder: Placeholder keeping the
-                dropout keeping probability
-
+            dropout_keep_prob: Probability of keeping neurons active in
```

Comment:
přerefaktoroval bys to ještě, aby to používalo neuralmonkey.nn.utils.dropout?

místo dropout placeholderu se udělá train_mode placeholder a pravděpodobnost se tomu předává jako float